app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    rename_case_flow = command.split(" ")
+    move_case_flow = command.split("/")
+
+    if rename_case_flow[0] != "mv":
+        raise ValueError("Wrong command")
+
+    if "/" not in rename_case_flow[2]:
```

Comment:
If you want check by / use os.path.split
Because in some os path will be `\\` instead of `/`